p1.py

```diff
@@ -0,0 +1,49 @@
+# Characters in phone corresponding to numbers from 2 to 9
+charsInPhone = [['a','b','c'],       # 2
+['d','e','f'],                       # 3
+['g','h','i'],                       # 4
+['j','k','l'],                       # 5
+['m','n','o'],                       # 6
+['p','q','r','s'],                   # 7
+['t','u','v'],                       # 8
+['w','x','y','z']]                   # 9
+
+# Make sure that the input only consists of 2-9s
+def isValidInput(_str):
+    for s in _str:
+        if s.isdigit() and int(s) >= 2 and int(s) <= 9:
+            continue
+        else:
+            return False
+    return True
+
+# Convert the 2-9s to charsInPhone list index
+def getIndex(i_str):
+    return int(i_str) - 2
+
+# Perform DFS on the string given by the user
+# res: result string to be printed
+# _str: string provided by the user
+# i: index of _str that is to be operated on
```

Comment:
The code's comments are welcome. But do we need to write comments during the virtual onsite. I'm looking forward to discussing this later!